base/strings/types.jl

```diff
@@ -155,7 +155,7 @@ reverseind(s::SubString{String}, i::Integer) =
 """
     repeat(s::AbstractString, r::Integer)
 
-Repeat a string `r` times.
+Repeat a string `r` times. This can equivalently be accomplished by calling [`^`](@ref).
```

Comment:
Would be clearer to write:
```
by calling [`s^r`](@ref ^).
```
Ideally, we should cross-reference `^(::AbstractString, ::Integer)` ... @MichaelHatherly, does `@ref ^(::AbstractString, ::Integer)` work in Documenter?